docs/config.md

```diff
@@ -82,6 +82,7 @@ The exporter is used to output the telemetry.
 | `OTEL_EXPORTER_OTLP_TIMEOUT` | Maximum time the OTLP exporter will wait for each batch export. | `1000` (ms) |
 | `OTEL_EXPORTER_OTLP_PROTOCOL` | The OTLP expoter transport protocol. Supported values: `grpc`, `http/protobuf`. [1] | `http/protobuf` |
 | `OTEL_EXPORTER_ZIPKIN_ENDPOINT` | Zipkin URL. | `http://localhost:8126` |
+| `OTEL_DOTNET_AUTO_HTTP2UNENCRYPTEDSUPPORT_ENABLED` | Enables `[System.Net](http://system.net/).Http.SocketsHttpHandler.Http2UnencryptedSupport`. Required by OTLP over gRPC when instrumenting NET Core 3.x applications. See [the official Microsoft documentation](https://docs.microsoft.com/en-us/aspnet/core/grpc/troubleshoot?view=aspnetcore-6.0#call-insecure-grpc-services-with-net-core-client) for more details. | `false` |
```

Comment:
Should we clarify that it's only required for gRPC on .NET Core 3.x when not exporting over https, which I think is the recommended configuration when exporting to an on-host collector instance?

Reply:
@nrcventura 
Do you mean something like the following?

```
| `OTEL_DOTNET_AUTO_HTTP2UNENCRYPTEDSUPPORT_ENABLED` | Enables `[System.Net](http://system.net/).Http.SocketsHttpHandler.Http2UnencryptedSupport`. Required when instrumenting .NET Core 3.x applications and using an non-TLS gRPC OTLP endpoint. See [the official Microsoft documentation](https://docs.microsoft.com/en-us/aspnet/core/grpc/troubleshoot?view=aspnetcore-6.0#call-insecure-grpc-services-with-net-core-client) for more details. | `false` |
```

Reply:
Yes, something like that would be good. I just don't want to direct people to use that setting unless they really need to.